### PR TITLE
Part 2 of the Restrucutre

### PR DESCRIPTION
**Pull request by carchi8py**

### CODE REVIEW

File: internal/provider/example_resource_test.go [hunk 1]
```diff
@@ -40,10 +36,10 @@ import (
...
 //}
 
-func testAccExampleResourceConfig(configurableAttribute string) string {
-	return fmt.Sprintf(`
-resource "netapp-ontap_example" "test" {
-  configurable_attribute = %[1]q
-}
-`, configurableAttribute)
-}
+//func testAccExampleResourceConfig(configurableAttribute string) string {
```
Comment (suhasbshekar) [inline internal/provider/example_resource_test.go:39]: comments needed?

File: internal/provider/name_services/name_services_dns_resource_test.go [hunk 2]
```diff
@@ -1,8 +1,9 @@
-package provider
+package name_services_test
 
 import (
 	"fmt"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	ntest "github.com/netapp/terraform-provider-netapp-ontap/internal/provider"
```
Comment (suhasbshekar) [inline internal/provider/name_services/name_services_dns_resource_test.go:6]: ntest? might need other name for this?

File: internal/provider/protocols/protocols_san_lun-maps_resource.go [hunk 3]
```diff
@@ -39,7 +40,7 @@ type ProtocolsSanLunMapsResource struct {
 // ProtocolsSanLunMapsResourceModel describes the resource data model.
 type ProtocolsSanLunMapsResourceModel struct {
 	CxProfileName     types.String `tfsdk:"cx_profile_name"`
-	SVM               SVM          `tfsdk:"svm"`
+	SVM               svm.SVM      `tfsdk:"svm"`
```
Comment (suhasbshekar) [inline internal/provider/protocols/protocols_san_lun-maps_resource.go:43]: svm.SVM or svm.name?

Comment (suhasbshekar) [reply internal/provider/protocols/protocols_san_lun-maps_resource.go:43]: NM, got it

**Review by wenjun666 — APPROVED** [review]

**Review by suhasbshekar — CHANGES_REQUESTED** [review]

File: internal/provider/example_resource_test.go (hunk 1 above)
Comment (carchi8py) [reply internal/provider/example_resource_test.go:39]: THe new linter will not allow us to have any function that has no use. THe example file have no use so I'm comment them all out.

File: internal/provider/name_services/name_services_dns_resource_test.go (hunk 2 above)
Comment (carchi8py) [reply internal/provider/name_services/name_services_dns_resource_test.go:6]: what would you call it.?

Comment (suhasbshekar) [reply internal/provider/name_services/name_services_dns_resource_test.go:6]: resource_test?

**Review by suhasbshekar — APPROVED** [review]

File: internal/provider/name_services/name_services_dns_resource_test.go (hunk 2 above)
Comment (chuyich) [reply internal/provider/name_services/name_services_dns_resource_test.go:6]: Agreed with @suhasbshekar to have something meaningful word

**Pull request merged**